src/main/java/com/autotune/common/performanceProfiles/ResourceOptimizationOpenShift.java

```diff
@@ -0,0 +1,130 @@
+package com.autotune.common.performanceProfiles;
+
+import com.autotune.common.data.result.ContainerResultData;
+import com.autotune.common.data.result.DeploymentResultData;
+import com.autotune.common.data.result.ExperimentResultData;
+import com.autotune.common.data.result.GeneralInfoResult;
+import com.autotune.common.k8sObjects.*;
+import com.autotune.common.performanceProfiles.perfProfileInterface.PerfProfileInterface;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Method;
+import java.util.*;
+
+public class ResourceOptimizationOpenShift implements PerfProfileInterface {
```

Comment:
`ResouceOptimizationOpenShift` is an implementation of `PerfProfileInterface` and so should be inside that dir. Also please rename that dir to `PerformanceProfileInterface`